File: 구현/개인 파일/KHK/Alarm_activity.java
package com.example.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.content.Intent;
import android.view.View;
import android.widget.Toast;

public class Alarm_activity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void onClick_01(View v) {
        Intent intent_01 = new Intent(getApplicationContext(), Alarmset_activity.class);
        startActivity(intent_01);
    }
    public void onClick_02(View v) {
        Toast toast_01 = Toast.makeText(this, "알람이 해제되었습니다.", Toast.LENGTH_SHORT);
        toast_01.show();
    }
}

File: 구현/종합본/worldlist_2/app/src/main/java/com/example/worldlist_2/AlarmActivity.kt
package com.example.worldlist_2

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Toast

class AlarmActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.alarm_activity)
    }

    fun View.onClick_1() {
        val intent_01 = Intent(applicationContext, Alarmset_activity::class.java)
        startActivity(intent_01)
    }

    fun View.onClick_2() {
        val toast_01 = Toast.makeText(this@AlarmActivity, "알람이 해제되었습니다.", Toast.LENGTH_SHORT)
        toast_01.show()
    }
}